packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -548,6 +548,41 @@ describe('plugin-meetings', () => {
             assert.calledOnce(mockSendSlotManagerCtor);
           });
         });
+        describe('#beRightBack', () => {
+          it('should have #beRightBack', () => {
+            assert.exists(meeting.beRightBack);
+          });
+
+          beforeEach(() => {
+            meeting.meetingRequest.locusDeltaRequest = sinon.stub().resolves();
+          });
+
+          it('should enable #beRightBack with the correct data and return a promise', async () => {
```

Comment:
The test description says here that function should enable brb with the correct data, and then I'd expect that we will check here if we called the request with proper data on something similar, like in admit test cases, for example. What do you think, @edvujic? 
Otherwise, now I do not see how we check data correctness in test here.